content/articles/facilitate-media-queries-using-scss-mixins/index.md

```diff
@@ -0,0 +1,154 @@
+ 
+SASS and SCSS should not be confused with each other, despite the similarity in their fundamental structure and their functionality as it concerns CSS. SCSS is a higher version of SASS, with greater programming features and flexibility when handling CSS code. Thankfully, their distinctions can be seen easily in their different syntaxes and file extensions.
+
+To take advantage of a CSS preprocessor, one needs to install the CSS compiler on the web server. There are various means of doing this, but an easy method is to locate the appropriate SASS package for your operating system on this [GitHub page](https://github.com/sass/dart-sass/releases/tag/1.43.5) and download it. Then you can simply install it and add the directory to your PATH. The path here refers to the listed directories where the OS looks for programs. That’s all. 
+
+For further instructions or directions on the installation process, the official [sass website](https://sasslang.com) is always available.
+
+### STEP 2
+#### Creating the SCSS File
+SCSS mixins give the ability to produce reusable blocks of code which aids to avoid repetition, creating cleaner code and enabling easy maintainability. There is no need to lay out the individual rules with tons of breakpoints which can be time-consuming and break up your code. Plus, you can avoid going through the frustration of searching through the CSS files for specific elements. Hence, it makes sense to use SCSS, particularly its mixins, to handle the media queries.
+
+At this point, It is assumed that you have a sass compiler on your system and are ready to go. To use the mixins, you need to create a file titled `style.scss` and then place it in the root folder for your project. Then execute this code on the command prompt for the compiler to read the scss file.
+
+```bash
+sass --watch style.scss:style.css
+```
+This action automatically creates a new CSS file named style.css in the folder.
+
+Mixins will help you manage your media query by defining it in one location. So you can apply it as needed, and then sass enables you to compile it to the corresponding CSS code.
+
+Let’s open our `style.scss` file and input the necessary code to set up the mixins:
+
+```scss
+// We define the mixins for two major breakpoints mobile and tablet
+ 
+ // $breakpoint is the variable that can have several values
+@mixin devices ($breakpoint) { //the name of the mixin is devices
+  
+    @if $breakpoint==tablet {    
+    @media only screen and (max-width: 680px) {
+      @content;
+    }
+  }
+ 
+  @if $breakpoint==mobile {
+    @media only screen and (max-width: 481px) {
+      @content;
+    }
+  }
+}
+ 
+```
+
+The above block of code shows how mixins are mapped out and associated with names.
+
+### STEP 3
+#### Media queries using SCSS mixins
+
```

Comment:
This heading: `Media queries using SCSS mixins` doesn't match with the previous one. Please rename it to match the other one. eg. `Writing media queries using mixins`

Reply:
@geoffrey45 Changed the heading to match with the main title "Facilitating Media queries using SCSS mixins". Thank you

Reply:
@geoffrey45 Changed the heading to match with the main title "Facilitating Media queries using SCSS mixins". Thank you